### PR TITLE
add timeout to install and upgrade, fix upgrade waitReady

### PR DESCRIPTION
**Pull request by miriSch**
This should resolve https://github.com/manusa/helm-java/issues/170 and also add the timeout option to the upgrade command and fix the waitReady option for the upgrade command

### CODE REVIEW

**Comment by miriSch** [comment]
@manusa do you have any idea why those tests failed? On my machine all tests are running fine.

**Comment by manusa** [comment]
I'm going over the code very quickly, but it's hard to find the issue unless I check it locally.

Things that you might want to check.
Notice that the cluster is shared across all tests in the same suite.
Probably the fact that all charts have the same name might create conflicts and race conditions if there are multiple tests aimed to succeed.

File: helm-java/src/test/java/com/marcnuri/helm/HelmKubernetesTest.java
Comment (miriSch) [inline helm-java/src/test/java/com/marcnuri/helm/HelmKubernetesTest.java:1]: @manusa should the added tests be adjusted to reflect your changes on helm-java/src/test/java/com/marcnuri/helm/HelmKubernetesTest.java in https://github.com/manusa/helm-java/pull/205/files (set fullNameOverride, image, and tag)?

**Comment by miriSch** [comment]
@manusa thank you for looking into the test failures!

File: helm-java/src/test/java/com/marcnuri/helm/HelmKubernetesTest.java
Comment (manusa) [reply helm-java/src/test/java/com/marcnuri/helm/HelmKubernetesTest.java:1]: To make the test suite less flaky, they should, yes.

Comment (manusa) [reply helm-java/src/test/java/com/marcnuri/helm/HelmKubernetesTest.java:1]: thx

**Review by manusa — APPROVED** [review]
LGTM, thx!!!

**Pull request merged**